Automattic/pocket-casts-ios
# End of Year: Implements the epilogue story design notes

| 📘 Project: #374 | 🛫 Depends on: #564 |
|:---:|:---:|

This addresses the following items:
- Updates the replay button to:
  - Use a different font
  - Adjust paddings
  - Use the design replay icon
- Updates the spacing
- Highlights Pocket Casts in the text

This also:
- Changes how the interactive stories work to no longer need to duplicate the entire story view for it to have interactive elements

## Screenshots

### Design vs Implementation Comparison

<p align="center"><img src="https://user-images.githubusercontent.com/793774/205428600-053210d1-e465-4fbd-a503-b026a0e496e7.png" width="100%" /></p>


### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
|<img src="https://user-images.githubusercontent.com/793774/205428525-aaff60ff-f037-41ca-a79e-c31432cb0bc2.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205428596-1b1f222a-9359-4998-a393-3daa01f90d3e.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205428549-ec9ee2ba-e456-4cb8-9325-9241870bccf5.png" width="320" />|


## To test

1. Launch the app
2. Tap on the Profile Tab > End of Year Card
3. Tap to the end
4. ✅ Verify the view looks good and matches the designs
5. ✅ Verify the Replay button works
6. ✅ Verify the close button works
7. ✅ Repeat the steps on various screen sizes

## Checklist
